app/views/user_verification/steps/_review_answers.html.erb

```diff
@@ -0,0 +1,58 @@
+<h1 class="govuk-heading-l">Check your answers</h1>
+
+<h2 class="govuk-heading-m">Application details</h2>
+
+<% f.object.answers.tap do |answers| %>
```

Comment:
Does this need to wrap everything?

Reply:
It doesn't and actually less nesting is preferable